packages/core/build/build.tokens.ts

```diff
@@ -80,7 +86,9 @@ function convertCSSValue(token: Token, base = 20) {
     // hsl
     value = `hsl(${token.value[0]}, ${token.value[1]}%, ${token.value[2]}%)`;
   } else if (token.value instanceof Token) {
-    value = `var(--cds-${camelCaseToKebab(token.value.name)}, ${convertCSSValue(token.value)})`; // reference
+    value = `var(--cds-${camelCaseToKebab(token.value.name)}${
+      fallback ? `, ${convertCSSValue(token.value, 20, fallback)}` : ''
+    })`; // reference
```

Comment:
When generating the alias tokens we can omit the fallback value since it is defined within the same global file.